lexicons/fyi/frontpage/richtext/facet.json

```diff
@@ -0,0 +1,10 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.richtext.facet",
+  "defs": {
+    "main": {
+      "type": "union",
+      "refs": []
```

Comment:
For now an empty union because we haven't implemented richtext and don't plan to until this lexicon is rolled out.

Eventually it'll be like

```json
[
  "#bold",
  "#italic",
  "app.bsky.richtext.facet#mention",
  "app.bsky.richtext.facet#link"
]
```

Reply:
This is now unused after I removed richtext from this change, removing.